tests/unit/test_cli.py

```diff
@@ -0,0 +1,53 @@
+import click
+from click.testing import CliRunner
+from colin.cli.colin import check, list_checks, list_rulesets
+
+def _call_colin(fnc):
+    runner = CliRunner()
```

Comment:
I forgot we can test click like this -- we can make more checks like this as a next step. (e.g. version would be good)